fastai/data/external.py

```diff
@@ -10,13 +10,15 @@
 
 # Cell
 @lru_cache(maxsize=None)
-def fastai_cfg():
+def fastai_cfg() -> Config: # Provides Config object for fastai's `config.ini`, that contains default download paths for `'data', 'model', 'storage' and 'archive'`
```

Comment:
```suggestion
def fastai_cfg() -> Config: # Config that contains default download paths for `'data', 'model', 'storage' and 'archive'`
```
They already know it's a config object for fastai from the docstring